tutorials/physics/interpolation/advanced_physics_interpolation.rst

```diff
@@ -0,0 +1,173 @@
+Camera that realigns slightly every 1/10th of a second (at 10tps tick rate) will
+often be noticeable. You can get a much smoother result by moving the Camera each
+frame in ``_process``, and following an interpolated target manually.
+
+Manual Camera interpolation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Ensure the Camera is using global coordinate space**
+
+The very first step when performing manual Camera interpolation is to make sure the
+Camera transform is specified in *global space* rather than inheriting the
+transform of a moving parent. This is because feedback can occur between the
+movement of a parent node of a Camera and the movement of the Camera Node itself,
+which can mess up the interpolation.
+
+There are two ways of doing this:
+
+1) Move the Camera so it is independent on its own branch, rather than being a child of a moving object.
+
+.. image:: img/fti_camera_worldspace.webp
+
+2) Call :ref:`Node3D.top_level<class_Node3D_property_top_level>` and set this to ``true``, which will make the Camera ignore the transform of its parent.
+
+Typical example
+^^^^^^^^^^^^^^^
+
+A typical example of a custom approach is to use the ``look_at`` function in the
+Camera every frame in ``_process()`` to look at a target node (such as the player).
+
+But there is a problem. If we use the traditional ``get_global_transform()`` on a
+Camera "target" Node, this transform will only focus the Camera on the target *at
+the current physics tick*. This is *not* what we want, as the Camera will jump
+about on each physics tick as the target moves. Even though the Camera may be
+updated each frame, this does not help give smooth motion if the *target* is only
+changing each physics tick.
+
+get_global_transform_interpolated()
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+What we really want to focus the Camera on, is not the position of the target on
+the physics tick, but the *interpolated* position, i.e. the position at which the
+target will be rendered.
+
+We can do this using the :ref:`Spatial.get_global_transform_interpolated<class_Node3D_method_get_global_transform_interpolated>`
+function. This acts exactly like getting :ref:`Spatial.global_transform<class_Node3D_property_global_transform>`
+but it gives you the *interpolated* transform (during a ``_process()`` call).
+
+.. important:: ``get_global_transform_interpolated()`` should only be used once or
+	           twice for special cases such as Cameras. It should **not** be used
+               all over the place in your code (both for performance reasons, and
+			   to give correct gameplay).
+
+.. note:: Aside from exceptions like the Camera, in most cases, your game logic
+	      should be in ``_physics_process()``. In game logic you should be calling
+          ``get_global_transform()`` or ``get_transform()``, which will give the
+		  current physics transform (in global or local space respectively), which
+		  is usually what you will want for gameplay code.
```

Comment:
```suggestion
.. important:: ``get_global_transform_interpolated()`` should only be used once or
               twice for special cases such as Cameras. It should **not** be used
               all over the place in your code (both for performance reasons, and
               to give correct gameplay).

.. note:: Aside from exceptions like the Camera, in most cases, your game logic
          should be in ``_physics_process()``. In game logic you should be calling
          ``get_global_transform()`` or ``get_transform()``, which will give the
          current physics transform (in global or local space respectively), which
          is usually what you will want for gameplay code.
```
Mixed tabs and spaces here, the notes were rendering funny:
![msedge_ph1OlCLv5Z](https://github.com/user-attachments/assets/0a3fd670-bc3a-4a45-9bc0-b36a081932d3)